runtime/JavaScript/package.json

```diff
@@ -20,27 +34,29 @@
   },
   "homepage": "https://github.com/antlr/antlr4",
   "devDependencies": {
-    "@babel/core": "^7.19.1",
-    "@babel/eslint-parser": "^7.19.1",
-    "@babel/preset-env": "^7.19.4",
-    "@types/node": "^18.7.23",
-    "babel-loader": "^8.2.5",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.3",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/preset-env": "^7.20.2",
+    "@rollup/plugin-babel": "^6.0.3",
+    "@types/node": "^18.15.3",
     "c8": "^7.12.0",
-    "compression-webpack-plugin": "^10.0.0",
-    "eslint": "^8.23.1",
-    "eslint-webpack-plugin": "^3.2.0",
-    "glob-parent": "^6.0.2",
-    "jasmine": "^4.0.2",
+    "eslint": "^8.36.0",
+    "jasmine": "^4.6.0",
     "jasmine-spec-reporter": "^7.0.0",
-    "minimist": "^1.2.6",
-    "source-map-support": "^0.5.21",
-    "terser-webpack-plugin": "^5.3.6",
-    "typescript": "^4.8.3",
-    "webpack": "^5.75.0",
-    "webpack-cli": "^4.10.0"
+    "del-cli": "^5.0.0",
+    "rollup": "^3.20.0",
+    "typescript": "^4.9.5"
   },
   "scripts": {
-    "build": "webpack",
+    "build-commonjs": "babel --config-file ./babel.commonjs.config.js --extensions \".js\" --out-dir lib ./src/antlr4",
```

Comment:
commonjs should be cjs and build into cjs, not into lib

Reply:
ok, I can change it, but you do recall it was just `dist` before anything else. And I took `es` and `lib` from common examples from other packages, but sure I can change to to `esm` and `cjs` if you prefer.

Reply:
it was dist when there was a single build. We don't want users to think that cjs is the recommended way of consuming antlr

Reply:
ok, I'll change it all to `lib` -> `cjs` and `es` -> `esm`

Reply:
thanks